README.md

```diff
@@ -78,6 +78,9 @@ $ ocic pod create --config test/testdata/sandbox_config.json
 ```
 
 #### Run a container inside a pod
+Note: The config for the following container specifies a mount for `host_path`
+at `/test/redis-data`. Please make sure this directory exists before running the
```

Comment:
I think we should just get rid of the mount from the config. We can add it to another config for testing.